SOYB-35 Adjust content styling within the iframe

sacarney opened the pull request:
Increase text size to be more legible, make lists format properly

jrearick commented:
Ahh, can't read a diff of a minified file very well. Was there a tool that you used to re-minify? Does the production site even use the minified version? Otherwise looks ok to me

sacarney commented:
Yea, I did it manually.... If you look at the unminified file, I added the new CSS to the end. Then copied it into the minified file and removed the whitespace by hand.

jrearick commented:
Works nicely locally. Merged to development. Will be in master next time I do a luggage release. The commit for the luggage_postcases_player submodule on soybeanresearch can be updated to f44ae1d29988e90532a0cfc1ff5d832526e40bc9 at any time
